src/main/java/teammates/ui/pagedata/InstructorSearchPageData.java

```diff
@@ -197,15 +197,13 @@ private void setSearchStudentsTables(StudentSearchResultBundle studentSearchResu
             String sectionName = student.section;
             String viewPhotoLink = addUserIdToUrl(student.getPublicProfilePictureUrl());
             emailToPhotoUrlMap.put(student.email, viewPhotoLink);
-            if (!teamNameToStudentsMap.containsKey(teamName)) {
-                teamNameToStudentsMap.put(teamName, new ArrayList<StudentAttributes>());
-            }
-            teamNameToStudentsMap.get(teamName).add(student);
-            if (!sectionNameToTeamNameMap.containsKey(sectionName)) {
-                sectionNameToTeamNameMap.put(sectionName, new ArrayList<String>());
-            }
-            if (!sectionNameToTeamNameMap.get(sectionName).contains(teamName)) {
-                sectionNameToTeamNameMap.get(sectionName).add(teamName);
+
+            teamNameToStudentsMap.computeIfAbsent(teamName, (String key) -> new ArrayList<>()).add(student);
+
+            List<String> teamNamesInSection = sectionNameToTeamNameMap.computeIfAbsent(sectionName, (String key) ->
+                                                                                                    new ArrayList<>());
```

Comment:
Prefer higher-level breaks to lower-level breaks.
https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html

There are multiple occurrences in your PR.